src/views/section.ts

```diff
@@ -48,41 +44,19 @@ export class SectionView extends BlockView<SectionNode> {
     this.dom.appendChild(this.contentDOM)
   }
   public onUpdateContent = () => {
-    const sectionTitleState = sectionTitleKey.getState(this.view.state)
-    const { titleSuppressed, generatedLabel, pageBreakStyle, id, category } =
-      this.node.attrs
-    const classNames: string[] = []
+    const sectionTitles = sectionTitleKey.getState(this.view.state)
 
-    if (titleSuppressed) {
-      classNames.push('title-suppressed')
-    }
-
-    if (typeof generatedLabel === 'undefined' || generatedLabel) {
-      classNames.push('generated-label')
```

Comment:
Are we sure we don't need that anymore? This is not manuscripts legacy thing, I remember developing it as a part of LWF implementation track.